spec/filters_spec.rb

```diff
@@ -173,6 +173,14 @@ def a_file(path)
         it "filters vendor/bundle" do
           expect(SimpleCov.filtered([a_file("vendor/bundle/foo.rb")]).count).to eq(0)
         end
+
+        it "filters hidden folders" do
+          expect(SimpleCov.filtered([a_file(".semaphore-cache/lib.rb")]).count).to eq(0)
+        end
+
+        it "filters hidden files" do
+          expect(SimpleCov.filtered([a_file(".hidden_config.rb")]).count).to eq(0)
+        end
```

Comment:
Could you please add a test to verify that we're not filtering hidden sub folders? I like your argument for not filtering so I think we should spec it as expected behaviour :)

Reply:
You're totally right! I added the test.